src/util/system.h

```diff
@@ -295,6 +295,9 @@ extern ArgsManager gArgs;
  */
 bool HelpRequested(const ArgsManager& args);
 
+/** Add help options to the args manager */
+void SetupHelpOptions(ArgsManager& args);
```

Comment:
nit, how this as a member, like `ArgsManager::SetupHelpOptions`?

Reply:
Agree with @MarcoFalke here. It is not the responsibility of `ArgsManager` to know about or hardcode specific options.